src/components/application_manager/test/sdl_preloaded_pt.json

```diff
@@ -366,6 +366,7 @@
                         "deviceStatus",
                         "engineOilLife",
                         "engineTorque",
+                        "climateData",
```

Comment:
Why do we need to update this copy of preloadedPT by hand instead of editing of CMakeLists.txt to copy actual preloadedPT automatically for all proposals of this type? I think that this work is redundant, isn't it?

Reply:
@ychernysheva great idea will check the suggested approach

Reply:
I think that we can omit this change due to bd4de14